app/controllers/katello/api/v2/hosts_bulk_actions_controller.rb

```diff
@@ -12,6 +12,7 @@ class Api::V2::HostsBulkActionsController < Api::V2::ApiController
     before_action :find_readable_hosts, :only => [:applicable_errata, :installable_errata, :available_incremental_updates]
```

Comment:
Do you happen to know why both ```:find_editable_hosts``` and ```find_readable_hosts``` are needed for applicable and installable errata? Might be an opportunity to fix that. 

```
    before_action :find_editable_hosts, :except => [:destroy_hosts, :applicable_errata, :installable_errata, :resolve_traces]
    before_action :find_deletable_hosts, :only => [:destroy_hosts]
    before_action :find_readable_hosts, :only => [:applicable_errata, :installable_errata, :available_incremental_updates]
```

Reply:
It seems totally unnecessary - removed it